### DISCUSSION TITLE
Separate "Configuration/Users" and "Server Admin/Users" pages are very confusing

### DISCUSSION THREAD

**Discussion by foogod** [grafana/grafana#42880] [General]
I just spent a fair bit of time pulling my hair out because I had not realized that there are actually two different places to configure information about users in Grafana, which cover overlapping but not identical information and operations, look very similar to each other, and are even named deceptively similarly in menus, etc.

I had a need to change another user's password, so I found https://grafana.com/docs/grafana/latest/manage-users/server-admin/server-admin-manage-users/ and was trying to follow it, but for some reason my "Users" page didn't look like the article's "Users" page.  It listed all the same users, but there were fields and options missing.  Is this doc for an older version of Grafana?  Am I logged into the wrong account?  No, it's just different for no apparent reason.

Eventually, when I read things *very* carefully, I came upon the single line "Hover your cursor over the Server Admin (shield) icon until a menu appears".  Wait, isn't the Users page under Configuration?  So I hover over "Server Admin" and, lo and behold, there's a completely different "Users" link to a completely different page, to allow me to manage the same users, but in different ways.

This is extremely confusing for anyone relatively new to Grafana, particularly since it does not appear to be really mentioned anywhere in the documentation I came across, except as an off-hand thing as if everyone already knows all about it.  I can't see any good reason why there should be two separate locations for this stuff, but if there must be can we at least make their names and appearances look more noticeably different from each other, and maybe actually call out that there are two places for this stuff in the documentation?

(IMHO a far better approach would be to just consolidate them into one location anyway, though)

**Environment**:
- Grafana version: Grafana v8.2.5 (b57a137acd)
- Data source type & version: N/A
- OS Grafana is installed on: Ubuntu 20.04.3 LTS
- User OS & Browser: MacOS / Brave Version 1.29.81 Chromium: 93.0.4577.82 (Official Build) (x86_64)
- Grafana plugins: N/A
- Others: N/A

**Comment by ashharrison90** [comment]
hey @foogod 👋 

thanks for raising this. i'm going to move this to a discussion for now so we can discuss what changes we think we'd like to make to help with the UX here. 

hope that makes sense! 😄

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
